oops/backplane/border.py

```diff
@@ -137,3 +136,32 @@ def border_outside(self, backplane_key):
 Backplane._define_backplane_names(globals().copy())
 
 ################################################################################
+
+
+
+################################################################################
+# UNIT TESTS
+################################################################################
+
+import unittest
+from oops.unittester_support           import TESTDATA_PARENT_DIRECTORY
+from oops.constants                    import DPR
+from oops.backplane.unittester_support import show_info
+
+
+#*******************************************************************************
+class Test_Border(unittest.TestCase):
+
+    #===========================================================================
+
+    def runTest(self):
```

Comment:
There should never be a blank line between a horizontal separator and a def. Remove.